### PR TITLE
Adds PR and Issue template.

### PR DESCRIPTION
**Pull request by SweptWasTaken**
This is what it looks like when you click "New Issue" (But of course with the edits i've made, this is goons one.)

image

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!-- The text between the arrows are comments - they will not be visible on your PR. -->
+<!-- To automatically tag this PR, add the uppercase label(s) surrounded by brackets below, for example: [LABEL] -->
```
Comment (ZeWaka) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: 🤔

File: .github/ISSUE_TEMPLATE/issue_report.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!-- To automatically tag this issue, add the uppercase label(s) surrounded by brackets below, for example: [LABEL] -->
```
Comment (ZeWaka) [inline .github/ISSUE_TEMPLATE/issue_report.md:1]: 🤔

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (SweptWasTaken) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: What exactly is the issue? Emojis aren't really helpful.

File: .github/ISSUE_TEMPLATE/issue_report.md (hunk 2 above)
Comment (SweptWasTaken) [reply .github/ISSUE_TEMPLATE/issue_report.md:1]: What exactly is the issue? Emojis aren't really helpful.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ZeWaka) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: You don't have auto labeling functionality...

File: .github/ISSUE_TEMPLATE/issue_report.md (hunk 2 above)
Comment (ZeWaka) [reply .github/ISSUE_TEMPLATE/issue_report.md:1]: You don't have auto labeling functionality... read your own PR text?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (PaulRitter) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: the idea is probably to add them especially for the autolabeling functionality

**Pull request merged**